### PR TITLE
Added some vocables.

### PR DESCRIPTION
**Pull request by Kaligule**
We are a bunch of algebraists who had fun with that.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -88,6 +104,9 @@
     ["Dynkin system"],
     ["sigma-algebra"],
     ["ultrafilter"],
+    ["Cayley-graph"],
```
Comment (alangpierce) [inline index.html]: I think this should be written without the hyphen? e.g. https://en.wikipedia.org/wiki/Cayley_graph

File: index.html [hunk 2]
```diff
@@ -113,6 +132,11 @@
     ["Markov chain"],
     ["4-form"],
     ["7-chain"],
+    ["operator"],
+    ["homeomorphism"],
+    ["colour"],
```
Comment (alangpierce) [inline index.html]: I think I'd prefer American english, so "color". (It looks like the math community is as uncertain about the spelling as the rest of the world, but let me know if I'm wrong and math people are more consistent.)

**Comment by alangpierce** [comment]
Thanks! I left some nitpicks, but happy to merge once those are addressed.

**Comment by Kaligule** [comment]
You are right.
- The Cayley graph was just misspelled.
- We choose the british spelling "colour" mostly by random, which somehow proofs your point of inconsistency :)

**Comment by alangpierce** [comment]
Thanks for the additions! Changes are live now.

**Comment by Kaligule** [comment]
:)

**Pull request merged**